src/utils/messages.cc

```diff
@@ -118,6 +118,12 @@ std::string legacyCLIStr( const std::string & old_r, const std::string & new_r,
        _("Legacy commandline option %1% detected. This option is ignored."))
        % NEGATIVEString(dashdash(old_r));
     break;
+  case LegacyCLIMsgType::Abbreviated:
+    return str::Format(
+       _("Abbreviated commandline option %1% will not be supported in future versions. Please use %2% instead.") )
```

Comment:
Same as above, restructuring the sentence to use the plural of "option" would be less ambigous